doc/nrf/releases_and_maturity/migration/nRF54H20_migration_2.7/migration_guide_2.4.99-cs3_to_2.7_application.rst

```diff
@@ -74,6 +74,34 @@ General
 
      The previously used SOC1-based board files have been removed.
 
+CoreMark
+--------
+
+Several changes have been made to migrate the CoreMark sample from the nRF54 customer sampling release v2.4.99-cs3 to |NCS| v2.7.0:
+
+* The sample has been moved from the Zephyr repository to the nRF Connect SDK repository.
+* Build system has been aligned to the Hardware Model v2.
+  The ``SB_CONFIG_IMAGE_2_BOARD`` and ``SB_CONFIG_IMAGE_3_BOARD`` Kconfig options have been replaced with the ``SB_CONFIG_APP_CPUNET_RUN`` and ``SB_CONFIG_APP_CPUPPR_RUN`` Kconfig options.
+  New Kconfig options more clearly indicates on which cores the sample will be run.
+  See the :file:`sysbuild.cmake` file to find out how these options are used to determine the board target.
+* Due to the fact, that the |NCS| v2.7.0 does not support STM logging for the nRF54, the STM logging has been removed from the CoreMark sample.
+  The sample now uses usual UART logging which allows for sending logs from only one core per UART instance.
+  The nRF54 has only two UART instances, so the sample now can be run on two cores at most.
+  The sample is always run on application core and depending on configuration can be run on either the radio core or the PPR core.
+  See the ``SB_CONFIG_APP_CPUNET_RUN`` and ``SB_CONFIG_APP_CPUPPR_RUN`` Kconfig options for more details.
+* The DTS overlays has been aligned to the |NCS| v2.7.0.
+* The :file:`system_nrf.h` library has been included explicitly in the :file:`main.c` file to print the CPU frequency.
+* The ``SB_CONFIG_PARTITION_MANAGER`` Kconfig option has been disabled in the :file:`sysbuild.conf` file to avoid using the partition manager.
+* The :kconfig:option:`CONFIG_APP_MODE_FLASH_AND_RUN` Kconfig option has been made promptless and enabled for the PPR core.
+  The PPR core is currently unable to access buttons and thus the :kconfig:option:`CONFIG_APP_MODE_FLASH_AND_RUN` Kconfig option must be enabled for this core to be able to run the benchmark.
+* To make the Coremark sample run on the PPR core, following build time arguments must be passed to the build system:
+  ``-DSB_CONFIG_APP_CPUNET_RUN=n -DSB_CONFIG_APP_CPUPPR_RUN=y -Dcoremark_SNIPPET=nordic-ppr-xip``.
+  The ``coremark_SNIPPET`` argument needs to be set to make the application core start the PPR core.
+  Alternatively, instead of adding those arguments manually, the sample can be built from the :file:`sample.yaml` file using following command:
```

Comment:
```suggestion
  Alternatively, instead of adding those arguments manually, the sample can be built from the :file:`sample.yaml` file using the following command:
```